core/modules/system/system.install

```diff
@@ -3123,6 +3123,31 @@ function system_update_1072() {
   // Empty update to force a rebuild of hook_library() and JS aggregates.
 }
 
+/**
+ * Add values for 403/404 enabled settings.
+ */
+function system_update_1073() {
+  $config = config('system.core');
+
+  $site_403 = $config->get('site_403');
+  if ($site_403 != '') {
+    $config->set('site_403_enabled', 1);
+  }
+  else {
+    $config->set('site_403_enabled', 0);
+  }
```

Comment:
I know that we're not going for the "shortest code to achieve things" award, but how about this?:

```suggestion
  $site_403_enabled = $config->get('site_403') != '' ? 1 : 0;
  $config->set('site_403_enabled', $site_403_enabled);
```